certified-connectors/PDFco/apiDefinition.swagger.json

```diff
@@ -0,0 +1,727 @@
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://pdf.co/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://pdf.co/pages/legal/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Productivity;Data"
+    }
+  ],
+  "host": "api.pdf.co",
+  "basePath": "/",
+  "schemes": ["https"],
+  "consumes": [
+    "application/json",
+    "application/x-www-form-urlencoded"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v1/pdf/convert/from/html": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK, success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "",
+                  "title": "url",
+                  "x-ms-visibility": "important"
+                },
+                "pageCount": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "pageCount",
+                  "x-ms-visibility": "important"
+                },
+                "error": {
+                  "type": "string",
+                  "description": "",
+                  "title": "error",
+                  "enum": ["", "true", "false"],
+                  "x-ms-visibility": "important"
+                },
+                "status": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "status",
+                  "x-ms-visibility": "important"
+                },
+                "name": {
+                  "type": "string",
+                  "description": "",
+                  "title": "name",
+                  "x-ms-visibility": "advanced"
+                },
+                "credits": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "credits",
+                  "x-ms-visibility": "advanced"
+                },
+                "duration": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "duration",
+                  "x-ms-visibility": "advanced"
+                },
+                "remainingCredits": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "remainingCredits",
+                  "x-ms-visibility": "advanced"
+                }
+              }
+            }
+          }
+        },
+        "summary": "HTML to PDF",
+        "description": "Convert HTML code snippet into full featured PDF.",
+        "operationId": "HTML_to_PDF",
```

Comment:
@hiren-byte operationId properties should use camel casing, an example can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers). Please fix operationId properties.